app/src/main/java/com/andrew/tasky/presentation/edit/EditFragment.kt

```diff
@@ -53,9 +53,11 @@ class EditFragment : Fragment(R.layout.fragment_edit) {
                     when (it) {
                         EditType.DESCRIPTION -> {
                             binding.editTypeTitle.text = getString(R.string.edit_description_text)
+                            binding.inputEditText.textSize = 16F
                         }
                         EditType.TITLE -> {
                             binding.editTypeTitle.text = getString(R.string.edit_title_text)
+                            binding.inputEditText.textSize = 26F
```

Comment:
Don't hardcode your text sizes in pixels, use SP units. These scale together with the user's font size preference. Android offers functions to convert these to pixels.